bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -121,6 +123,7 @@ public AbstractBigtableAdmin(CommonConnection connection) throws IOException {
     disabledTables = connection.getDisabledTables();
     bigtableInstanceName = options.getInstanceName();
     tableAdapter = new TableAdapter(bigtableInstanceName);
+    this.instanceName = InstanceName.of(bigtableInstanceName.getProjectId(), bigtableInstanceName.getInstanceId());
```

Comment:
Please get the instance name from `bigtableInstanceName.toGcbInstanceName()`, if it exists.

Reply:
Apparently `v2.models.CreateTableRequest` expects `com.gogle.bigtable.admin.v2.InstanceName`.
and `bigtableInstanceName.toGcbInstanceName()` returns `v2.models.InstanceName`.

Reply:
Please note that I will be removing Names from the google-cloud-java to avoid these kind of headaches.
What is toGcbInstanceName?

Reply:
@igorbernstein2, We have a `BigtableInstanceName` in this repo.  `toGcbInstanceName` converts the `BigtableInstanceName` to an `InstanceName`.  See [here](https://github.com/GoogleCloudPlatform/cloud-bigtable-client/commit/82b9bb6db4e861161f164cd5e514ec2d8f9e96c8#diff-36056d6784a87e2537c8ba1864c4d9f2R125).